components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceEndpointConfig.java

```diff
@@ -126,6 +127,8 @@
     @UriParam
     private String apexMethod;
     @UriParam
+    private String compositeMethod;
```

Comment:
Can this be used both on consumer or producer? If not then add label to mark it as consumer or producer only.

Reply:
Like `apexMethod` it's consumer only for now. Let me improve this (we should update `apexMethod` as well but it's another PR).